R/orderly.R

```diff
@@ -66,8 +66,10 @@ orderly_run <- function(name, parameters = NULL, envir = NULL,
   parameters <- check_parameters(parameters, dat$parameters)
   inputs <- c("orderly.yml", dat$script, dat$resources, dat$sources)
 
+  plugins <- intersect(names(root$config$plugins), names(dat))
```

Comment:
Any value in erroring here if plugin declared in `plugins` but then has no configuration? or has configuration but not declared in plugins? Maybe more fiddly than is worth

Reply:
* If a plugin has no configuration, that could be ok (perhaps it does not need one?)
* If a plugin uses configuration but is not listed we won't know it was a plugin, but it will error in the existing field check

Reply:
oh nice! That sounds good